clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/JobDescription.java

```diff
@@ -540,6 +550,10 @@ public JobDescriptor getGrpcJobDescriptor() {
       containerBuilder.addEntryPoint(entryPoint);
     }
 
+    if (cmd != null) {
```

Comment:
@aravindmd I think this might just need to be a not null or empty check here

Reply:
Thank @cfieber , looks like that is the case , i'll fix it.